lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -347,357 +280,387 @@
+          right: 0;
           width: 100%;
-          min-height: 100%;
         }
       }
 
-      .info {
-        @include clearfix();
-        @include padding(0, 10px, 0, 230px);
+      .wrapper-course-details {
+        display: block;
+        @include float(left);
+        width: flex-grid(9);
+        padding: 0;
+        height: 120px;
+      }
 
-        > hgroup {
-          padding: 0;
-          width: 100%;
+      .course-title {
+        border-bottom: 2px solid $gray-l5;
 
-          .university {
-            color: $lighter-base-font-color;
-            font-family: $sans-serif;
-            font-size: 16px;
-            font-weight: 400;
-            margin: 0 0 6px;
-            text-transform: none;
-            letter-spacing: 0;
-          }
+        a, span {
+          @extend %t-title3;
+          @extend %t-light;
+          display: block;
+          margin-bottom: ($baseline/2);
 
-          .date-block {
-            position: absolute;
-            top: 0;
-            @include right(0);
-            font-family: $sans-serif;
-            font-size: 13px;
-            font-style: italic;
-            color: $lighter-base-font-color;
+          &:hover, &:focus {
+            text-decoration: none;
           }
+        }
+      }
 
-          h3 a, h3 span {
-            display: block;
-            margin-bottom: ($baseline/2);
-            font-family: $sans-serif;
-            font-size: 34px;
-            line-height: 42px;
-            font-weight: 300;
+      .course-info {
+        display: block;
+        @include float(left);
+        width: flex-grid(4);
+        padding: 0;
+        margin-top: ($baseline/2);
 
-            &:hover, &:focus {
-              text-decoration: none;
-            }
-          }
+        [class*="info-"] {
+          color: $gray-d1;
+          @extend %t-title6;
+          display: inline-block;
         }
 
-        .course-status {
-          background: $yellow;
-          border: 1px solid $border-color-2;
-          box-shadow: 0 1px 0 0 rgba(255,255,255, 0.6);
-          margin-top: 17px;
-          margin-right: flex-gutter();
-          padding: ($baseline/4);
-          width: flex-grid(8);
-          float: left;
-          @include box-sizing(border-box);
-
-          p {
-            color: $lighter-base-font-color;
-            font-style: italic;
-            letter-spacing: 1px;
-            text-align: center;
-          }
+        .info-date-block {
+          @extend %t-title7;
+          color: $gray-l1;
+          display: block;
         }
+      }
 
-        .course-status-completed {
-          background: $gray-l3;
-          color: $very-light-text;
+      .wrapper-course-actions {
+        display: block;
+        @include float(right);
+        width: flex-grid(8);
+        padding: 0;
+        margin-top: ($baseline/2);
+      }
 
-          p {
-            color: #222;
+      .course-actions {
+        position: relative;
+        top: 50%;
```

Comment:
that's a great question actually. I think this is a carry over from earlier (or from some work I wound up cleaning up). I've removed these styles now in a new commit. thanks for flagging it!